docs/accessibility.md

```diff
@@ -279,10 +279,7 @@ if (Platform.OS === 'android') {
 
 To enable TalkBack, go to the Settings app on your Android device or emulator. Tap Accessibility, then TalkBack. Toggle the "Use service" switch to enable or disable it.
 
-P.S. Android emulator doesn’t have TalkBack by default. To install it:
-
-1. Download TalkBack file here: https://google-talkback.en.uptodown.com/android
-2. Drag the downloaded `.apk` file into the emulator
+Android emulators don't have TalkBack installed by default. You can install TalkBack on your emulator via the Google Play Store. Make sure to choose an emulator with the Google Play store installed. These are available in Android Studio.
```

Comment:
I like the idea of using a screenshot as @blavalla mentioned highlighting the column that shows which emulators have Play store installed. Do you think that makes sense?

Reply:
I think it makes sense! I'm not sure how to add an image to the website though (plus make it mobile friendly and whatnot), I'd have to look into how to do that.

Reply:
Yea.. good point. I found this usage of an image on "Handling Touches": https://reactnative.dev/docs/handling-touches
with source here: https://raw.githubusercontent.com/facebook/react-native-website/67f71d97500da48de34bced6b35ceddac07738ab/website/versioned_docs/version-0.64/handling-touches.md

Might be able to copy that... not sure if it's been mobile optimized either or different asset sizes etc. Might be worth asking in the OSS chat if anyone has advice. Feel free to drop and add another time if it's too involved, accepting to unblock!

Reply:
I'll file a task to add this later!